conditional/templates/dashboard.html

```diff
@@ -251,15 +251,15 @@ <h3 class="panel-title">Major Projects</h3>
             </div>
             {% endif %}
 
-        </div>
+            {% if student and not active %}
+                <div class="alert alert-info"><span class="glyphicon glyphicon-exclamation-sign white" style="padding-right:5px"></span>
+                    Hey there! You're eligible to become an active member, click the button below if you'd like to pay dues and become active!
```

Comment:
Hey there, you're eligible to become an active member! Click the button below if you'd like to become active and pay dues.